website/docs/compaction.md

```diff
@@ -74,22 +67,47 @@ spark-submit --packages org.apache.hudi:hudi-utilities-bundle_2.11:0.6.0 \
 --continous
 ```
 
-### Hudi CLI
-Hudi CLI is yet another way to execute specific compactions asynchronously. Here is an example
+## Executing Async Compaction
+After compactions have been asynchronously scheduled on the Hudi Timeline, you can now execute those compactions with one of the following options
```

Comment:
Related to above comments - When using Deltastreamer and Spark Streaming we dont need to use these tools separately to execute compactions.

Reply:
I guess the discussion I had with @kywe665 caused the confusion here.  In Hudi Kafka Connect Sink, the compaction/clustering is only scheduled and it's upon these separate utilities to do the actual services (I'll also see if this gap should be bridged).

In terms of Deltastreamer, from the existing docs, it was not clear to me if it does both scheduling and execution as I was reading the docs at the first glance.  We should call that out.

Reply:
Thanks for catching this @bhasudha, yes I think @yihua and I mixed up two conversations. I reverted the changes so it stays as scheduled and executed together